snippets/csharp/VS_Snippets_CLR/process_sample/CS/process_sample.cs

```diff
@@ -17,60 +17,58 @@
 using System.Diagnostics;
 using System.Threading;
 
-namespace Process_Sample
+namespace ProcessSample
 {
-   class MyProcessClass
-   {
-      public static void Main()
-      {
-         try
-         {
-
-            Process myProcess;
-            myProcess = Process.Start("NotePad.exe");
-
-            while(!myProcess.HasExited)
+    class MyProcessClass
+    {
+        public static void Main()
+        {
+            try
             {
-               Console.WriteLine();
+                using (Process myProcess = Process.Start("NotePad.exe"))
+                {
+                    while (!myProcess.HasExited)
+                    {
+                        Console.WriteLine();
 
-               // Get physical memory usage of the associated process.
-               Console.WriteLine("Process's physical memory usage: " + myProcess.WorkingSet);
-               // Get base priority of the associated process.
-               Console.WriteLine("Base priority of the associated process: " + myProcess.BasePriority);
-               // Get priority class of the associated process.
-               Console.WriteLine("Priority class of the associated process: " + myProcess.PriorityClass);
-               // Get user processor time for this process.
-               Console.WriteLine("User Processor Time: " + myProcess.UserProcessorTime);
-               // Get privileged processor time for this process.
-               Console.WriteLine("Privileged Processor Time: " + myProcess.PrivilegedProcessorTime);
-               // Get total processor time for this process.
-               Console.WriteLine("Total Processor Time: " + myProcess.TotalProcessorTime);
-               // Invoke overloaded ToString function.
-               Console.WriteLine("Process's Name: " + myProcess.ToString());
-               Console.WriteLine("-------------------------------------");
+                        // Get physical memory usage of the associated process.
```

Comment:
These also would be good candidates for string interpolation.

Reply:
These may be too long.